frontend/src/renderer/InputHandling/handlers/VariableIntensityHandler.ts

```diff
@@ -32,36 +34,55 @@ const VariableIntensityHandler = (): InputHandler => {
       });
 
       const triggerActions = binding.actions.filter(isTriggerAction);
-      if(triggerActions.length > 0) {
-        if(binding.activeTriggers > 0) {
-          const groupedActions: { [key: string]: number[] } = {}
-          triggerActions.forEach(action => {
-            if(groupedActions[action.name] === undefined) {
-              groupedActions[action.name] = [action.channel];
-            } else {
-              groupedActions[action.name].push(action.channel);
-            }
-          })
+      if (triggerActions.length > 0) {
+        //1 is pressed
+        //2 is held
+        //0 is released/ not held
+        const groupedActions: { [key: string]: number[] } = {}
+        triggerActions.forEach(action => {
+          if (groupedActions[action.name] === undefined) {
+            groupedActions[action.name] = [action.channel];
+          } else {
+            groupedActions[action.name].push(action.channel);
+          }
+        })
+
+        Object.entries(groupedActions).forEach(([name, channels]) => {
+          let li = lastIntensity.get(name) as Map<number, number>
+          if (li == undefined) lastIntensity.set(name, new Map<number, number>())
+          li = lastIntensity.get(name) as Map<number, number>
+
+          //only update channels that have differing intensities in compairson to last iteration
+          const filteredChannels: number[] = []
+          channels.forEach(c => {
+            const int = li.get(c)
+            if (int == undefined) li.set(c, binding.activeTriggers == 0 ? 0 : intensities[name])
+            if (
+              (int as number > intensities[name] + sendThreshold) ||
+              (int as number < intensities[name] - sendThreshold)
```

Comment:
Um das Typing einzuhalten (damit wir nicht as number machen). Wer weiß was JS bei undefined > number machen würde :sweat_smile: 
```suggestion
            let int = li.get(c)
            if (int == undefined) {
              int = binding.activeTriggers == 0 ? 0 : intensities[name];
              li.set(c, int);
            }

            if (
              (int > intensities[name] + sendThreshold) ||
              (int < intensities[name] - sendThreshold)
```